src/router/index.js

```diff
@@ -16,20 +16,25 @@ const routes = [{
         query: {
             src: ''
         },
-        props: route => ({ experiment: route.query.experiment })
-
+        props: route => ({ 
+            experiment: route.query.experiment,
+            id: route.params.id 
+        })
     },
     {
         // type: the type of component invoking this path (optional)
         // id: the unique ID for the component invoking this path (optional)
         path: "/login/:id?/:type?",
         name: "Phone Sign In",
         component: PhoneSignIn,
+        props: true
```

Comment:
https://router.vuejs.org/guide/essentials/passing-props.html#passing-props-to-route-components